analytics-api-core/src/test/scala/org/ekstep/analytics/api/util/TestPostgresDBUtil.scala

```diff
@@ -55,7 +55,8 @@ class TestPostgresDBUtil extends FlatSpec with Matchers with BeforeAndAfterAll {
     
     new GeoLocationCity();
     new GeoLocationRange();
-    new ReportConfig()
-    EmbeddedPostgresql.close();
+    new ReportConfig();
+    new JobRequest();
+//    EmbeddedPostgresql.close();
```

Comment:
Any reason why is this commented out?

Reply:
Other postgres dependent test cases execution after executing TestPostgresDBUtil test cases were failing with **org.postgresql.util.PSQLException: FATAL: terminating connection due to administrator command** error and removing close from here resolved the error.